packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -366,6 +368,30 @@ export class CallContext {
     });
   }
 
+  /* @conditional-compile-remove(ppt-live) */
+  public setCallPPTLiveActive(callId: string, isActive: boolean): void {
+    this.modifyState((draft: CallClientState) => {
+      const call = draft.calls[this._callIdHistory.latestCallId(callId)];
+      if (call) {
+        call.pptLive.isActive = isActive;
+      }
+    });
+  }
+
+  /* @conditional-compile-remove(ppt-live) */
+  public setCallParticipantPPTLive(callId: string, target: HTMLElement | undefined): void {
+    this.modifyState((draft: CallClientState) => {
+      const call = draft.calls[this._callIdHistory.latestCallId(callId)];
+      const participantKey = call.htmlShareRemoteParticipant;
```

Comment:
nit: we usually refer to the id of the participant as `participantId` this usually is used in place of the flat version of the identifier which maps to the type `CommunicationIdentifier`